docs/content/usages/events.md

```diff
@@ -4,37 +4,72 @@ sort: 6
 
 # Events API
 
-A Nikita session extends the [native Node.js Events API](https://nodejs.org/api/events.html). It provides a facility to listen to internal notification and know the current state of the program.
+A Nikita session extends the [native Node.js Events API](https://nodejs.org/api/events.html). It provides a facility to listen to internal notifications and know the current state of the program.
 
 ## Usage
 
-Listening to events with the Node.js API is quite simple with the `on(event, handler)`. The "handler" argument is the function provided by the user to catch events. 
+Listening to events with the extended Node.js API is quite simple with the `tools.events.on(event, handler)` function available as an argument of the [action's handler](/current/action/handler). The `handler` argument is the function provided by the user to catch events.
 
-It is not recommended to call the `emit` function directly but instead your are encouraged to call the `nikita.log` function which will associate the "type" property with the event of the same name.
+To emit an event, it is not recommended to call the `emit` function directly but instead, you are encouraged to call the `tools.log()` function which will associate the `type` property with the event of the same name. 
+
+The following example demonstrates a combination of listening and emitting an event:
+
+```js
+nikita(({tools}) => {
+  // Register a listener
+  tools.events.on('my_event', (log) => {
+    console.info(`I am ${log.message}`)
+  })
+  // Emit an event
+  tools.log({
+    message: 'Nikita',
+    type: 'my_event'
+  })
+})
+// Outputs with "I am Nikita"
+```
```

Comment:
This page is about event, not about logging. Log shall be covered in a separate page as we will add more to it (I am planning for `log.info` for example)

Reply:
Moved